ubuntu-20.04SOC/scripts/cleanup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eux
+
+# Apt cleanup.
+apt autoremove
+apt update
+
+# Delete unneeded files.
+rm -f /home/vagrant/*.sh
+rm -rf /home/vagrant/netman/netman_1.0.2_linux_64-bit.zip /home/vagrant/netman/netman_1.0.2_linux_64-bit
+rm -rf /home/vagrant/gwireguard*
+rm -rf /home/vagrant/gip_*.zip
+
+# Zero out the rest of the free space using dd, then delete the written file.
+#dd if=/dev/zero of=/EMPTY bs=1M
```

Comment:
Not sure what this command is doing. I receive no space left when it is not comment, at building time

Reply:
https://serverfault.com/questions/677021/what-is-dd-if-dev-zero-of-empty-bs-1m

Reply:
https://serverfault.com/questions/677021/what-is-dd-if-dev-zero-of-empty-bs-1m